Gruntfile.js

```diff
@@ -54,7 +54,17 @@ module.exports = function(grunt) {
         options: {
           jshintrc: '.jshintrc'
         },
-}
+      },
+
+      concat: {
+          options: {
+            separator: ';',
+          },
+          dist: {
+            src: ['node_modules/jquery/dist/jquery.min.js', 'node_modules/leaflet/dist/leaflet.js', 'src/object.js'],
```

Comment:
Then concat also seems redundant with Browserify and uglify - which are we using to compile?  It looks liek concat, based on line 76, but maybe we should be using Browserify, no?